tools/getnodereportversion.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import print_function
+import os
+import re
+
+def get_version():
+  package_json = os.path.join(
+    os.path.dirname(__file__),
+    '..',
+    'deps',
+    'node-report',
+    'package.json')
+
+  f = open(package_json)
+
+  regex = '^\s*\"version\":\s*\"([\d.]+)\"'
+
+  for line in f:
+    m = re.match(regex, line)
+    if m:
+      return m.group(1)
+  
+  raise Exception('Could not find pattern matching %s' % regex)
+
+if __name__ == '__main__':
+  print(get_version())
```

Comment:
as suggested earlier, if we are porting natively, can we eliminate this altogether, and record the version statically?

Reply:
In my opinion once this is in core and there is no longer a link to the module node-report doesn't require a version because it is no longer a dependency (i.e. it versions with the version of Node.js that is released).